src/seedsigner/views/psbt_views.py

```diff
@@ -128,22 +131,27 @@ def run(self):
             else:
                 num_self_transfer_outputs += 1
 
-        screen = psbt_screens.PSBTOverviewScreen(
-            spend_amount=psbt_parser.spend_amount,
-            change_amount=psbt_parser.change_amount,
-            fee_amount=psbt_parser.fee_amount,
-            num_inputs=psbt_parser.num_inputs,
-            num_self_transfer_outputs=num_self_transfer_outputs,
-            num_change_outputs=num_change_outputs,
-            destination_addresses=psbt_parser.destination_addresses,
-        )
+        spend_amount = psbt_parser.spend_amount
+        change_amount = psbt_parser.change_amount
+        fee_amount = psbt_parser.fee_amount
+        num_inputs = psbt_parser.num_inputs
+        destination_addresses = psbt_parser.destination_addresses
```

Comment:
Minor note: doesn't seem to be any reason to pull these out into local vars when the values can just be directly passed in below in lines 147-153.